core/src/main/java/org/infinispan/persistence/spi/InitializationContext.java

```diff
@@ -74,4 +75,9 @@
     * Returns the persistence marshaller which should be used to marshall/unmarshall all stored bytes.
     */
    PersistenceMarshaller getPersistenceMarshaller();
+
+   /**
+    * Returns the global state configuration so that file-based stores can use default locations for storage
+    */
+   GlobalStateConfiguration getGlobalStateConfiguration();
```

Comment:
Maybe we should return the `GlobalConfiguration` here to allow greater flexibility? Some of the store implementations, jdbc for sure, have to retrieve the global config via the cache -> CacheManager currently.

Reply:
Done